docker/README.md

```diff
@@ -0,0 +1,86 @@
+# Description
+This Docker Compose development environment includes
+
+* PHP 5.6.*
+* MySQL 5.6
+* Nginx 1.6.2
```

Comment:
Adding ElasticSearch is as easy as adding the following lines:
```
elasticsearch:
    image: elasticsearch
    environment:
        DNSDOCK_ALIAS: elasticsearch.sylius.docker
```
Should this be added in a default development environment?

Git is available in the `php` container, but I don't think it's good practice to use git inside a container while developing. It's required for `composer`.